pkgs/by-name/wa/walker/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGo122Module,
+  fetchFromGitHub,
+  gtk4,
+  gtk4-layer-shell,
+  gobject-introspection,
+  pkg-config,
+}:
+
+buildGo122Module rec {
```

```suggestion
buildGoModule rec {
```

`buildGoModule == buildGo122Module` is in latest Nixpkgs.